assets/app/scripts/controllers/overview.js

```diff
@@ -238,6 +239,7 @@ angular.module('openshiftConsole')
       if (configJson) {
         try {
           var depConfig = $.parseJSON(configJson);
+          // FIXME: Can this go away?
```

Comment:
We shouldn't be overwriting `status` on the deployment (which is a replicationcontroller) in either case, or we'll lose the live `replicas` count the system keeps up to date. The point of this is to add in deploymentconfig details, so we had trigger information about the deployment.

Change this to
```
deployment.details = depConfig.details || depConfig.status.details;
```

And change back the bits in https://github.com/openshift/origin/commit/7138704972cf8f0d97fe89334d33ae623da7b90f?w=1#diff-8a316f49faea19e66ae3f4b8518a4135L45 to look at `deployment.details.causes...`